### DISCUSSION TITLE
SimpliSafe not allowing authentication login

### DISCUSSION THREAD

**Discussion by kjustie** [homebridge-simplisafe3/homebridge-simplisafe3#447] [Q&A]
I'm setting up Homebridge on a new server (and want to start from scratch rather than copying from existing server), and can't get the SimpliSafe website login page to let me login:  Mac Safari, Mac Chrome, and iOS Safari all say all cookies are blocked, but they aren't; I've confirmed settings, restarted browsers, etc.  I can login in at the website's home page on Mac and iPhone, but not at the auth login URL from the plugin.  And even after logging in at the home page, the auth login still wants me to log in again (and fails).   The SimpliSafe mobile URL I would have pasted into the settings on my existing server isn't in the JSON Config that I can see, nor in the plugin's settings, so I can't copy it from the old server, so...
-- Is there a way to find that URL somewhere on my existing server, in order to copy it to the new server (assuming that would work)?
or
-- any suggestions for getting the Simplisafe site to recognize that I don't have cookies blocked, so I can log in and complete the auth process the usual way?
(The last time I had to do this was in March, and it worked fine then.)
thanks!

**Comment by shamoon** [comment]
I think someday SS will break the auth here, hopefully not any time soon… But no, I’m not aware of anything like this so I’m not sure what’s happening. I assume it has to do with the auth layer SS use (not your browser etc).

Anyway, inside your homebridge “storage” folder there’s a file called `simplisafe3auth.json` that you can move over and if the token hasn’t expired you’ll be able to move it over…

**Reply by kjustie** [reply]
That worked -- thanks!
kevin

**Comment by Antbdp** [comment]
Hi Shamoon, I am trying the step in Chrome and have accepted 3rdparty cookies, opened the JavaScript console, launched the login from the plugin, authenticated, and completed the MFA. When
 I look at the error message in the console, I cannot find the com.SimpliSafe.mobile:// URL being referred to, can you have a look at the attached and provide some pointers if I'm int he right place or not, and or if I'm doing something wrong
[Simplisafe plugin.pdf](https://github.com/user-attachments/files/19032240/Simplisafe.plugin.pdf)
[Simplisafe plugin.pdf](https://github.com/user-attachments/files/19032243/Simplisafe.plugin.pdf)

**Reply by shamoon** [reply]
Please don’t comment on someone else’s discussion, start your own. That screenshot shows a url with “suspensions” in it which would make me wonder if your account is… suspended

**Reply by Antbdp** [reply]
Apologies, i started my own discussion. can you comment there, please
